lib/taskjuggler/Project.rb

```diff
@@ -361,6 +362,8 @@ def initialize(id, name, version)
               true,  false,   false, KeywordArray.new([ '*' ]) ],
         [ 'end',       'End',          DateAttribute,
               true,  true,    false, nil ],
+        [ 'ref',       'Ref',          DateAttribute,
```

Comment:
The keyword 'ref' is too generic. Please use 'markdate' or a similarly telling keyword.